### DISCUSSION TITLE
Copilot crashes and cannot be uninstalled or updated

### DISCUSSION THREAD

**Discussion by txdywmwddhxs** [community/community#39059] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Originally, the copilot could work. I just drank water and then crashed. The IDE displays this LOGO. 

![image](https://user-images.githubusercontent.com/56292188/201642912-f88f409b-c270-4725-b259-ef1fe4c5bcd2.png)

When I tried to update or uninstall copilot, copilot asks me to restart the ide. But when I restart, it shows this.

![image](https://user-images.githubusercontent.com/56292188/201642487-94f2f6ea-5b86-410a-9be2-9f6d5ac8c14a.png)

![image](https://user-images.githubusercontent.com/56292188/201642563-488816d9-55ad-4822-a1c0-100a9075b74f.png)

What should I do? I really loved copilot...  Even now.

**Comment by txdywmwddhxs** [comment]
Besides, I tried re-login copilot, but it still doesn't work

**Comment by wosk0x01** [comment]
Try reinstall the hole IDE when u cant find out a solution (i gonna been painful since u needs to re-config everything)
i solve a similar problem by reinstall

**Reply by txdywmwddhxs** [reply]
I reinstalled Goland. When I install copilot, ide displayed this. 

![image](https://user-images.githubusercontent.com/56292188/201647020-11802407-f8e7-4f8a-ad5d-f16c379001d7.png)

**Comment by iamcjwilliams** [comment]
@txdywmwddhxs Do you have the latest version?

**Reply by txdywmwddhxs** [reply]
![image](https://user-images.githubusercontent.com/56292188/201807813-abbb8c68-94c9-4199-8f70-c20c7c16b22f.png)

I think this is the latest version. But it cannot be installed.

**Reply by iamcjwilliams** [reply]
Which IDE are you using?

**Reply by txdywmwddhxs** [reply]
Goland 2021.3.3

**Comment by txdywmwddhxs** [comment, marked as answer]
I swear i didn't do anything! But I retried to install copilot this morning! This time it works!
